src/main/webapp/invalid.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="utf-8"/>
+      <link rel="icon" href="/favicon.ico"/>
+      <meta name="viewport" content="width=device-width,initial-scale=1"/>
+      <meta name="theme-color" content="#000000"/>
+      <meta name="description" content="Web site created using create-react-app"/>
+      <link rel="apple-touch-icon" href="/logo192.png"/><link rel="manifest" href="/manifest.json"/>
+      <title>Invalid Dataset Creation</title>
+      <link href="/static/css/main.09051bff.chunk.css" rel="stylesheet">
+    </head>
+  <body>
+    <h1>Invalid Dataset Creation</h1>
+    <p>The dataset you attempted to create already exists</p>
+    <p>Try Again: <a href="/index.html">Retry</a></p>
+  </body>
+</html>
```

Comment:
Pls add extra line here and to all other files.

Reply:
Will do! Changed that

Reply:
I still see missing new lines (e.g., in asset-manifest.json).

Reply:
That was also automatically generated, so everytime I run the app, i believe it won't have the new line